kesirler ikinci soru

```diff
@@ -0,0 +1,45 @@
+using System;
+
+
+namespace ConsoleApp35
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double sayi1, sayi2, top, cık, carp, bol;
+            char islem;
+            Console.Write("sayi 1 giriniz: ");
+            sayi1 = Convert.ToInt16(Console.ReadLine());
+            Console.Write("sayi 2 giriniz: ");
+            sayi2 = Convert.ToInt16(Console.ReadLine());
+            top = sayi1 + sayi2;
```

Comment:
Demo scope u olabildindigince kucuk tut dayicim. 
Top islem == 1 in icine girsin bosu bosuna hesaplama.